.travis.yml

```diff
@@ -6,13 +6,6 @@ php:
   - 7.0
 
 dist: trusty
-sudo: required
```

Comment:
The `sudo apt-get -y install mysql-server` statement lower down works just fine :)